pladder/bot.py

```diff
@@ -67,31 +72,37 @@ def __init__(self, state_dir):
         self.register_command("help", self.help)
         self.register_command("version", self.version)
 
-    def RunCommand(self, text):
-        return self.interpret(text)
+    def RunCommand(self, timestamp, network, channel, nick, text):
+        context = {'timestamp': timestamp,
```

Comment:
Wouldn't it be nice if this was a datetime.datetime object when it reaches the commands? Could be a separate change, though.

Reply:
Example: `datetime.fromtimestamp(timestamp, tz=timezone.utc)`

Maybe we want this datetime object to be in local time? Then it would be: `datetime.fromtimestamp(timestamp, tz=timezone.utc).astimezone(tzlocal.get_localzone())` . I haven't thought this through....

Reply:
Done, sticking with UTC for now